mod.ts

```diff
@@ -165,6 +166,10 @@ export interface BuildOptions {
   };
   /** Action to do after emitting and before running tests. */
   postBuild?: () => void | Promise<void>;
+  /** Override the log Info function, by default it's console.log */
+  loggerInfo?: (message: string) => void;
+  /** Override the log Warn function, by default it's console.log */
+  loggerWarn?: (message: string) => void;
```

Comment:
I'd rather not expose these because it doesn't apply to subprocesses.

Reply:
yep, any suggestions?

I really not an expert in logging flavor, my opinion, is that console.log `works` but is bad, So I also use it until I get complaints about them.

my recent log experiences were:
- using and improving an external lib: [x/logger](`https://deno.land/x/logger@v1.1.1`) I'm now maintaining this lib :)
- using a nodeJS EventEmitter to get broadcast my log.

----
by the way, currently, the console.log of the test is visible during the `deno run test`.

What is the official way to use EventEmitter-like feature in the Deno core team? EventEmmiter was dropped in std 0.177, and I do not know why.
I would have preferred to emit events, for each step of dnt process.